.github/workflows/ci.yml

```
@@ -55,23 +72,23 @@ jobs:
           npm-i: mocha@9.2.2
           
         - name: Node.js 15.x
-          node-version: "15.14"
+          node-version: "15.14.0"
```

That made me wonder, should we really be testing against the latest versions of old Node.js majors? What if we break compat with 15.0.0 by accident?

---

```suggestion
          node-version: "15"
```
yup, this should a nice change.